atomics/T1127/T1127.yaml

```diff
@@ -17,3 +17,25 @@ atomic_tests:
     elevation_required: false
     command: |
      C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe #{filename}
+
+- name: MSBuild Bypass Using Custom Tasks
+  description: |
+    Executes the code in a project file and local assebmly using. C# Example
+    Also demonstrates how to pass inputs to Custom Tasks.
+  supported_platforms:
+    - windows
+  input_arguments:
+    source_code:
+      description: Location of source code for custom Task
+      type: Path
+      default: PathToAtomicsFolder\T1127\src\T1127.cs
+    filename:
+      description: Location of the project file
+      type: Path
+      default: PathToAtomicsFolder\T1127\src\T1127CustomTask.csproj
+  executor:
+    name: command_prompt
+    elevation_required: false
+    command: |
+      C:\Windows\Microsoft.NET\Framework\v4.0.30319\csc.exe /r:Microsoft.Build.Framework.dll,Microsoft.Build.Tasks.v4.0.dll,Microsoft.Build.Utilities.v4.0.dll,System.IO.Compression.dll /target:library /out:PathToAtomicsFolder\T1127\src\T1127.dll /out:PathToAtomicsFolder\T1127\src\T1127.cs
```

Comment:
Is it expected to have two "/out:" parameters in this command such that the T1127.cs file gets overwritten when you run the test? Preventing you from running the test again . . .

Reply:
P.S. thank you for you "first-time" contribution. We love to see more people join in!

Reply:
Thats a typo, let me clean this up and reissue.